src/app/projects/pages/projects-list/projects-list.component.html

```diff
@@ -104,7 +104,8 @@
         </td>
 
         <td class="vf-table__cell">
-          <p>{{ project.cellCount ? (project.cellCount | number) : "-" }}</p>
+          <span *ngIf="project.cellCount">{{ (project.cellCount | number) }}</span>
+          <p *ngIf="!project.cellCount">-</p>
```

Comment:
```suggestion
          <span *ngIf="project.cellCount; else noCellCount">{{ (project.cellCount | number) }}</span>
         <ng-template #noCellCount>-</ng-template>
```

Reply:
It needs to be a paragraph or a span element in each case. I think your suggestion actually adds more lines. Normally, I would go for this if we didn't need different elements